lib/compiler/passes/generate-bytecode.js

```diff
@@ -564,13 +585,15 @@ function generateBytecode(ast) {
 
     literal(node) {
       if (node.value.length > 0) {
-        const stringIndex = addConst("\""
+        const match = node.match | 0;
+        const needConst = match === 0 || (match > 0 && !node.ignoreCase);
+        const stringIndex = needConst ? addConst("\""
```

Comment:
are you really attempting to manually quote string escaping?

Reply:
Again, that PR not tries to change everything in one commit. Escaping problem is solving in the #117

Reply:
It turns out the first line is a substitute for `(if node.?match === 0)`

It's still not clear why match is being controlled with arithmetic, despite asking several times, because I got some monk on the mountain "when you truly understand what match is" to a direct question "why are you doing this"

The third one can easily incur side effects during maintenance because of the over-tricky use of ternary select, where the correct form is actually `if` 

This should not land.  We are not creating a trivia contest.

Reply:
> ... to a direct question "why are you doing this"

As I say, introduction comment coming soon